### ISSUE TITLE
loss divergence

### ISSUE THREAD

**Issue by zhuoxizeng1212** [GeorgeChenZJ/deepcount#2]
Hi thank you for your great work and sharing it with the community. I am trying to re-train your model with a resnet backbone, training from scratch. However, it seems like model would not converge at all.

Have you experienced anything similar to this with VGG backbone?

**Comment by GeorgeChenZJ** [comment]
Hi. For a network with batch norm layers such as resnet, I believe a larger batch size would help (experience suggests a good start is from 64).  Other training parameters matter too.
